context/app/static/js/components/home/Hero/const.tsx

```diff
@@ -13,6 +13,20 @@ const timelineIconProps = {
 } as const;
 
 export const HOME_TIMELINE_ITEMS: TimelineData[] = [
+  {
+    title: 'Unified Dataset Pages Now Available',
+    titleHref: '/search?entity_type[0]=Dataset',
+    description: 'Datasets and their processed descendants are now displayed on a single page.',
```

Comment:
Is descendant a term users would be immediately familiar with?

Reply:
Updated text:

```Dataset page design has been updated to display both raw and processed data on the same page, enhancing the understanding of relationships between datasets.```

Reply:
Updated text for title and description are available here: https://hms-dbmi.atlassian.net/wiki/spaces/GL/pages/3536748545/Homepage+Latest+Changes+Entries